packages/disploy/src/structs/managers/StructureManager.ts

```diff
@@ -4,7 +4,7 @@ import { Base } from '../Base';
 
 export class StructureManager<T extends Base> extends Base {
 	public constructor(app: App, private object: NonRuntimeClass<T>, private _fetch: (id: string) => Promise<unknown>) {
-		super(app);
+		super(app, {});
```

Comment:
In a future PR when `.raw` is implemented, we should make a new class called `BaseDiscordStructure` which would be `Base` in its current state, and `Base` would just have the `App` class attached to it.